### PR TITLE
Highlight authors

### PR DESCRIPTION
**Pull request by danielwilliams94**
**What:**
Add functionality to contentful highlights to be able to render an author underneath their body

**Why**
So that highlights can contain quotes/customer testimonials with an attribution

### CODE REVIEW

File: src/components/Contentful/Highlight.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ export default function ContentfulHighlight({
   html,
   style,
   textAlign,
+  author,
```
Comment (mmmmillar) [inline src/components/Contentful/Highlight.js]: Can we keep the params in alphabetical order please

File: src/components/Contentful/_contentful_highlight.scss [hunk 2]
```diff
@@ -72,4 +72,17 @@
       }
     }
   }
+
+  .contentful-highlight__author {
+    span {
+        height: 45px;
+        font-size: 14px;
```
Comment (mmmmillar) [inline src/components/Contentful/_contentful_highlight.scss]: font-size and line-height need to be in rems and these should already exist in the contentful_settings scss. also, the properties with value "normal" don't need to be defined.

File: src/components/Contentful/index.js [hunk 3]
```diff
@@ -79,6 +79,7 @@ function ComponentRenderer(content) {
           html={documentToHtmlString(content.body && content.body.json)}
           style={content.style}
           textAlign={content.textAlign}
+          author={content.author}
```
Comment (mmmmillar) [inline src/components/Contentful/index.js]: alphabetical

**Comment by danielwilliams94** [comment]
I've addressed your comments @mmmmillar

**Pull request merged**